core/src/core.cpp

```diff
@@ -213,7 +205,11 @@ bool yukarin_s_forward(int length, long *phoneme_list, long *speaker_id, float *
     Ort::Value output_tensor = to_tensor(output, phoneme_shape);
 
     status->yukarin_s.Run(Ort::RunOptions{nullptr}, inputs, input_tensors.data(), input_tensors.size(), outputs,
-                           &output_tensor, 1);
+                          &output_tensor, 1);
+
+    for (int i = 0; i < length; i++) {
+      if (output[i] < 0) output[i] = PHONEME_LENGTH_MINIVAL;
```

Comment:
[MAY] めったに起こらないことですが、0から微小値の間だった場合はそのままになるのは意図通りですか？以下の方が直観的な気がします

```suggestion
      if (output[i] < PHONEME_LENGTH_MINIVAL) output[i] = PHONEME_LENGTH_MINIVAL;
```

Reply:
issueに書かれた要件に従ったためこのようにしましたが、そちらのほうが自然だと思いますので修正します